core/src/main/java/com/scalar/db/util/ActiveExpiringMap.java

```diff
@@ -83,6 +93,10 @@ public boolean contains(K key) {
     return map.containsKey(key);
   }
 
+  public boolean containsValue(V value) {
+    return map.values().stream().map(ValueHolder::get).anyMatch(value::equals);
```

Comment:
It looks to create a whole intermediate collection by `map`.
```
    return map.values().stream().anyMatch(holder -> value.equals(holder.get()));
```
Maybe this can avoid it?

Reply:
> t looks to create a whole intermediate collection by `map`.

Oh really? the `map()` method is an intermediate operation and also returns `Stream<R>`. Does it create a while intermediate collection?

Reply:
Oops, I may have confused with other language's one. Sorry for the confusion.